### DISCUSSION TITLE
How can I open files/web urls on agents?

### DISCUSSION THREAD

**Discussion by trmdi** [Ylianst/MeshCentral#5981] [Q&A]
Is it possible to open a file or a web url on an agent from the server?

**Comment by si458** [comment]
Yes, u click the little globe in the remote tab and type in the url and it should open the users default browser.

However only urls can be opened, not files

**Reply by trmdi** [reply]
Sorry I can't find it. Could you post a screenshot? Is it also available on the mobile webUI?

**Reply by si458** [reply]
No it's not available on the mobile ui, its only on the web ui
It's in the bottom left hand corner when u connect to a device, a tiny globe symbol

**Reply by trmdi** [reply]
I still can't find it. Could you please post the screenshot?
Edit: ah, found it. For some reason, it doesn't show when I browse on the mobile (display in the PC mode).

**Reply by si458** [reply]
Yes, the ui is best for mobile in mobile mode, don't switch to desktop mode, as stuff is hidden/missing, I've spotted that recently with my z fold 3

If u want the feature on the mobile ui, u need to open a feature request in the issues section

**Reply by trmdi** [reply]
https://github.com/Ylianst/MeshCentral/issues/5984
